networkit/graphio.pyx

```diff
@@ -552,14 +725,34 @@ cdef extern from "<networkit/io/EdgeListPartitionReader.hpp>":
 
 
 cdef class EdgeListPartitionReader:
-	""" Reads a partition from an edge list type of file
-	 """
+	""" 
+	EdgeListPartitionReader(firstNode=1, sepChar = '`\`t')
+	
+	Reads a partition from an edge list type of file.
+
+	Parameters
+	----------
+	firstNode : int, optional
+		Id of first node. Default: 1
+	sepChar : str
+		Character which is used for data seperation. Default: '\t'
+	"""
 	cdef _EdgeListPartitionReader _this
 
-	def __cinit__(self, node firstNode=1, sepChar = '\t'):
+	def __cinit__(self, node firstNode=1, sepChar = `'\t'`):
 		self._this = _EdgeListPartitionReader(firstNode, stdstring(sepChar)[0])
 
 	def read(self, path):
+		"""
+		read(path)
+		
+		Reads a file.
```

Comment:
Maybe expand a little bit (e.g., reads a partition from an edge list file).

Reply:
Good point - also added return-values for the read-function, since not all return a graph-datastructure.